crates/holochain/src/conductor/kitsune_host_impl.rs

```diff
@@ -99,10 +99,10 @@ impl KitsuneHost for KitsuneHostImpl {
     fn is_blocked(
         &self,
         input: kitsune_p2p_block::BlockTargetId,
-        timestamp: Timestamp,
+        timestamp: kitsune_p2p_types::Timestamp,
```

Comment:
Where we're interacting with Kitsune, I've had to continue to reference the Kitsune Timestamp type. These should sort themselves out as we start replacing this code.